CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -135,21 +108,26 @@ function Enable-DockerDriver {
         }
 
         # Nested ScriptBlock variable passing workaround
+        $OSCreds = $Using:OSCreds
         $AdapterName = $Using:AdapterName
         $ControllerIP = $Using:ControllerIP
-        $Configuration = $Using:Configuration
-        $TenantName = $Using:TenantName
 
         Start-Job -ScriptBlock {
-            Param ($Cfg, $ControllerIP, $Tenant, $Adapter)
-
-            $Env:OS_USERNAME = $Cfg.Username
-            $Env:OS_PASSWORD = $Cfg.Password
-            $Env:OS_AUTH_URL = $Cfg.AuthUrl
-            $Env:OS_TENANT_NAME = $Tenant
-
-            & "C:\Program Files\Juniper Networks\contrail-windows-docker.exe" -forceAsInteractive -controllerIP $ControllerIP -adapter "$Adapter" -vswitchName "Layered <adapter>" -logLevel "Debug"
-        } -ArgumentList $Configuration, $ControllerIP, $TenantName, $AdapterName | Out-Null
+            Param ($OpenStack, $ControllerIP, $Adapter)
```

Comment:
Maybe we should exclude this rule which checks variables if it includes credentials in PSScriptAnalyzer instead of renaming `OSCreds` to `Openstack`

Reply:
I've tried to suppress the message here, but without success. We've agreed with @katrybacka that we should just exclude this rule globally and from what I remember, she's planning to do it.

Anyway, I'm going to refactor this function in the next pull request, so I don't think it makes sense to change it here. Do you agree?

Reply:
Actually i added this comment, we were doing this review together without Mateusz. For me it is fine to change it in other pr, i was planning to do the excluding of this mentioned rule anyway.